template_config.yml

```diff
@@ -26,11 +26,13 @@ plugin_dash_short: ansible
 plugin_name: pulp_ansible
 plugin_snake: pulp_ansible
 plugin_snake_short: ansible
+publish_docs_to_pulpprojectdotorg: false
 pulp_settings:
   ansible_api_hostname: http://pulp:80
   ansible_content_hostname: http://pulp:80/pulp/content
-pulpcore_branch: master
-pulpcore_pip_version_specifier: null
+pulpcore_branch: 3.6
+pulpcore_pip_version_specifier: '>=3.6,<3.7'
```

Comment:
I manage to get the CI green with these values,
but 3.6 branch has `drf-spectacular==0.9.13`
and `pulpcore>=3.6,<3.7` has `drf-spectacular>=0.9.12`
which means at the release job it will break unless we release `pulpcore 3.6.4`